ui/src/main/js/model/runs-stage-grouped.js

```diff
@@ -181,14 +181,14 @@ function addStageTotals(runGroup) {
         var run = runGroup.runs[i];
 
         if (run.stages && (run.status === 'IN_PROGRESS' || run.status === 'PAUSED_PENDING_INPUT')) {
-            addCompletionEstimates(run, runGroup.avgDurationMillisNoPause, runGroup.runs.length);
+            addCompletionEstimates(run, runGroup.avgDurationMillisNoPause, endToEndRuns);
```

Comment:
*May* avoid factoring incomplete runs into completion estimates.  Possible fix for [JENKINS-38536](https://issues.jenkins-ci.org/browse/JENKINS-38536)